hack/common.sh

```diff
@@ -305,6 +305,7 @@ os::build::internal::build_binaries() {
         eval "platform_goflags=(${!platform_goflags_envvar:-})"
 
         GOOS=${platform%/*} GOARCH=${platform##*/} go install \
+          -pkgdir ${GOPATH}/src/${OS_GO_PACKAGE} \
```

Comment:
@tdawson @smarterclayton @stevekuznetsov FYI this line makes it so the go stdlib as well as all the origin & vendored code have their .a files placed in the root of the origin dir, so you get bufio.a, bytes.a, ...

@damemi in case you were wondering why this was happening